bitbots_hcm/src/hcm.cpp

```diff
@@ -149,26 +166,31 @@ class HCM_CPP : public rclcpp::Node {
   }
 
   void dynup_callback(const bitbots_msgs::msg::JointCommand msg) {
-    if (current_state_ == humanoid_league_msgs::msg::RobotControlState::STARTUP ||
-        current_state_ == humanoid_league_msgs::msg::RobotControlState::FALLEN ||
-        current_state_ == humanoid_league_msgs::msg::RobotControlState::GETTING_UP ||
-        current_state_ == humanoid_league_msgs::msg::RobotControlState::CONTROLLABLE) {
+    if (current_state_ == bitbots_msgs::msg::RobotControlState::STARTUP ||
+        current_state_ == bitbots_msgs::msg::RobotControlState::GETTING_UP ||
+        current_state_ == bitbots_msgs::msg::RobotControlState::MOTOR_OFF ||
+        current_state_ == bitbots_msgs::msg::RobotControlState::PICKED_UP ||
```

Comment:
Because we might want to play the sitdown down in case of an overload or play the walkready animation if we are picked up so we can be placed down nicely afterwards. The later is arguably more important, as we don't really perform any sit down motions due to the springs.